Dank Pull Request

Snepsts opened the pull request:
Merge my sick pull yo

Snepsts committed:
Rename guessgame to guessgame.py

Use the right file extensions you heathen

Snepsts committed:
Make it a guessing game

agunther97 commented:
+1

agunther97 committed:
Update README.md

Snepsts committed:
Merge pull request #1 from rocke97/patch-2

Update README.md

agunther97 committed:
Updated description comment

Made the comment reflect the true scope of the project

Snepsts committed:
Update README to reflect naming standards

Snepsts committed:
Merge pull request #2 from rocke97/patch-3

Updated description comment

Snepsts commented:
The pull request has become approx 69% danker, please consider merging

agunther97 commented:
Comments really expand on the scope, and frankly the beauty of the project.

abaga129 commented:
This is a really important feature that will improve the user's experience and other mumbo jumbo.  I also have no idea what I'm talking about.

**edit**
+1

smithandrewl commented:
I'm giving this pull request a rating of Aaron/Michael, which is really my dankest rating.

Snepsts committed:
Add attempts to the game to make the odds winning from 1/20 to 1/4

Snepsts commented:
Ok how about now, Ethan?

kaburkett committed:
really up the ante on the thrills

kaburkett committed:
I realize I made it unreasonable... after these changes it's possible to win

Snepsts committed:
Merge pull request #3 from kaburkett/master

really up the ante on the thrills

Snepsts commented:
Updated with even more sick changes

kaburkett commented:
+1

I can now comfortably say it's a crime if this PR is not merged soon. 

[mor important info her](http://www.asciify.net/ascii/ascii/6507)

+1 this post if you check out the important info

smithandrewl committed:
Implemented as a reusable class with a commandline demonstration program at the bottom. Paves the way for the GUI version.

Snepsts committed:
Merge pull request #4 from smithandrewl/reusable_class

Implemented as a reusable class

messej committed:
fixed final turn comparison

Now you know your true suckiness if you lose

messej committed:
Why use lambdas

you're just printing a string

Snepsts committed:
Merge pull request #5 from messej/master

fixed final turn comparison

Commit:
Add superior readme and scores

The other readme was far too lackluster. This one is better in everyway. Also there's now some variables to be used for a scoring system.

StaneCobalt committed:
Sneks hate tabs and globals

smithandrewl committed:
Added a basic tkinter GUI.

smithandrewl committed:
Uses ttk widgets and the clam theme.

smithandrewl committed:
Added glade designer file for UI.

smithandrewl committed:
Added a class taking a builder instance which handles UI events and updates the game.

smithandrewl committed:
Widget instances are loaded, and the game is updated whenever next is clicked.

smithandrewl committed:
Game status label updates according to game object state changes.

smithandrewl committed:
Game led icons update colors depending on how high or low you guessed.

smithandrewl committed:
Fixed logic error in attempt tracking.

smithandrewl committed:
Both led icons are red at game start.

smithandrewl committed:
Added a helper function to set the status lights.

smithandrewl committed:
led labels are updated when you when or lose.

smithandrewl committed:
Fixed logic error and disabled button after game is over.

smithandrewl committed:
Application exits when the window is closed.

smithandrewl committed:
Code cleanup.

smithandrewl committed:
Added doc comments.

smithandrewl committed:
Renamed the glade file.

smithandrewl committed:
The answer is now shown when you lose.

smithandrewl committed:
Updated comments and author.

smithandrewl committed:
Added to the readme.

smithandrewl committed:
Fixed image links in readme.

smithandrewl committed:
Added spaces after the headings.

smithandrewl committed:
Removed spaces from image names.

smithandrewl committed:
Updated readme.

smithandrewl committed:
Corrected file name in screenshots directory.

smithandrewl committed:
Added development info at the bottom.

smithandrewl committed:
Corrected screen shot captions.

smithandrewl committed:
Added a Contributors section to the readme

smithandrewl commented:
I finally finished the Gui version, it uses Gtk and Glade.
Screenshots  [here](https://github.com/smithandrewl/guessgame/tree/gui)
@rocke97

kaburkett commented:
Aw hell yeah *head bob*

On Sun, Sep 23, 2018, 12:55 AM Andrew Smith <notifications@github.com>
wrote:

> I finally finished the Gui version, it uses Gtk and Glade.
> Screenshots here <https://github.com/smithandrewl/guessgame/tree/gui>
> @rocke97 <https://github.com/rocke97>
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/EthanEdDall/guessgame/pull/2#issuecomment-423791969>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AMORPCypW03yQA4qI2eNNhTd_q9_5Rhiks5udxQogaJpZM4WkDf4>
> .
>

Snepsts committed:
Merge pull request #6 from StaneCobalt/master

Add superior readme and scores

Snepsts committed:
Merge branch 'master' into gui

Snepsts committed:
Fix name mistake

Snepsts committed:
Merge pull request #7 from smithandrewl/gui

The gui version has arrived.

Commit:
Add snek

StaneCobalt committed:
Add webpage and snek icon

StaneCobalt committed:
Update webpage

Gave it a dark theme. Also made text yellow and blue to fit the python color scheme.

Snepsts committed:
Merge pull request #8 from StaneCobalt/master

Snek logo and webpage

Snepsts committed:
Remove nohup.out and expand gitignore
